awards.md

```diff
@@ -19,34 +19,34 @@ An `epoch` is a fixed number of layers. Each Testnet epoch has XXX layers.
 
 So `Epoch 0` is the first XXX layers starting with `layer 0` and its last layer is `layer XXX-1`, and `Epoch 1` start in `Layer XXX` and ends with `Layer 2xXXX-1`.
 
-Your eligibility to contribute blocks to a specific layer is automatically determined by the Spacemesh protocol.
+Your eligibility to contribute blocks to a specific layer is determined by the Spacemesh protocol, and automatically executed by the App.
 
 The Spacemesh protocol guarantees that you will be eligible to submit at least one block in at least one layer in each and every epoch.
 
 ## Layer Awards
 Each layer has a total award of 1 Spacemesh Coins. This award is distributed to the awards account of each miner who contributed at least one block to that layer.
 
-So, for example, if a layer includes 10 blocks, each submitted by a different full node, 1 SMC will be distributed evenly between the coin awards accounts of each of the 10 participating full nodes. So each account will be credited with 0.1 SMC.
+So, for example, if a layer includes 10 blocks, each submitted by a different full node, 1 SMC will be distributed evenly between the coin awards accounts of each of the 10 participating full nodes. Each account will be credited with 0.1 SMC.
 
 ## Layer Awards Adjustment
 This award amount is adjusted by the number of transactions included in blocks. The awards for full nodes who have submitted less than 15 transactions in their submitted block is reduced by 15%. This penalty is distributed as a bonus evenly to all full nodes who have submitted more than 15 transactions in their submitted block in the layer.
 
-## Transaction Fees Awards
+## Transaction-Fee Awards
 In addition to the layer award described above, all full nodes who submitted at least one block to a layer are entitled for a share of the transaction fees of valid transactions included in blocks in that layer.
 
 The transaction fees in a layer, are evenly distributed to all the full nodes who have submitted a block in the layer.
 
 ## Tracking your Awards
-Awards are added your awards account when 5 layers have passed since a layer that you are entitled to receive an award  for.
+Awards are added your awards account when 5 layers have passed since a layer that you are entitled to receive an award for.
```

Comment:
Why 5? It should be much more (in Bitcoin I believe that it is 100)

Reply:
Yet another open testnet param to get a closure on quickly. The 5 is from @antonlerner